tools/report-converter/codechecker_report_converter/analyzers/analyzer_result.py

```diff
@@ -148,10 +149,13 @@ def _write(
         analyzer_info = AnalyzerInfo(name=self.TOOL_NAME)
         for file_path, file_reports in file_to_report.items():
             source_file = os.path.basename(file_path)
+            file_hash = hashlib.md5(file_path.encode(errors='ignore')) \
```

Comment:
It would be useful to normalize the file path above when we put it into the dictionary here:
https://github.com/Ericsson/codechecker/blob/fc3d1fe905b2bdd474939ccece8955c7b4e695c4/tools/report-converter/codechecker_report_converter/analyzers/analyzer_result.py#L146

Change this line to this:
```py
file_path = os.path.normpath(report.file.original_path)
file_to_report[file_path].append(report)
```

This way if we have `a/b/../x.cpp` and `a/x.cpp` it will not treat it as two different file paths and will generate two plist files but only one.

Reply:
I'm not clear on this concept there may be a typo in the last line but I made the change

Reply:
@milanlakhani In which line do you think there is a typo?

Reply:
it will not treat it as two different file paths and will generate two plist files but only one. @csordasmarton

Reply:
two plist files but one filepath? i don't see the advantage